webpack.common.js

```diff
@@ -1,7 +1,28 @@
 const path = require('path')
+const webpack = require('webpack')
+
 const HTMLWebpackPlugin = require('html-webpack-plugin')
 const FaviconsWebpackPlugin = require('favicons-webpack-plugin')
 
+const apikeys = require('./apikeys.js')
+
+if (!apikeys.graphhopper) {
+    const emptyApiKeys = {
+       "graphhopper": "missing_api_key",
+       "maptiler": "missing_api_key",
+       "omniscale": "missing_api_key",
+       "thunderforest": "missing_api_key",
+       "kurviger": "missing_api_key",
+    }
+
+    console.log('Missing apikeys.js file or missing graphhopper key inside this file. Get it from https://graphhopper.com/dashboard')
```

Comment:
This does not work, if `apikeys.js` is missing the require call will already fail (and the error message is not as clear), at least this is happened when I tried it.